shift-shift.go

```diff
@@ -86,6 +88,7 @@ func main() {
 			display, keyFirst, keySecond,
 			*printMode, *quietMode,
 			reDeviceMatch, *scanOnce,
+			*dblKeystroke, *dblKeyTimeout,
```

Comment:
Well, it has become a bad pattern after all. The function has got too many parameters (9 after this PR). It is not an issue for the current PR but it should be refactored later. I just left this comment as a todo for myself.